other/lru_cache.py

```diff
@@ -266,7 +266,7 @@ def get(self, key: T) -> U | None:
         self.miss += 1
         return None
 
-    def set(self, key: T, value: U) -> None:
+    def set_key(self, key: T, value: U) -> None:
```

Comment:
`set()` is still called in the doctests.

Reply:
> `set()` is still called in the doctests.

Yep, just made a commit to fix this